src/EditorFeatures/CSharpTest/AutomaticCompletion/AutomaticLineEnderTests.cs

```diff
@@ -3132,6 +3132,30 @@ public void Bar2()
 }");
         }
 
+        [WpfFact]
+        public void TestCheckedStatement()
```

Comment:
a test that verifies it also works in `unchecked` and one to verify that nothing happens for `var m = checked(1+1)` would be good to have I think

Reply:
Sure